help-beta/README.md

```diff
@@ -1 +1,7 @@
 # Help Center Beta Docs
+
+This directory is part of an ongoing migration project to convert our help center documentation to use starlight and add search to the docs at the same time.
+
+The status of the project can be tracked at this umbrella issue: https://github.com/zulip/zulip/issues/30450
+
+If you are a developer, you refer to `/devtools` page on your zulip installation to know what commands to use to run the project for development.
```

Comment:
You should line-wrap your documentation; I think we generally do 72 characters for Markdown files.

Reply:
Fixed!